src/accordion/accordion.ts

```diff
@@ -199,6 +207,14 @@ export class NgbAccordion implements AfterContentChecked {
    */
   isOpen(panelId: string): boolean { return this._states.get(panelId); }
 
+  /**
+   * @internal
```

Comment:
@jiayihu once again, the main problem with the `hidden` attribute is that ... you shouldn't use it at all!!! Please use appropriate Bootstrap 4 CSS classes instead, following Bootstrap's markup as documented officially: https://v4-alpha.getbootstrap.com/components/collapse/#accordion-example

BTW, we might need to merge #1332 first.